src/views/About/About.jsx

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import { formatVersion, isOlder } from '../../api/Version';
 import LoadingString from '../../components/common/LoadingString/LoadingString';
 import Row from '../../components/common/Row/Row';
 
 import logo from './logo.png';
 
 import styles from './About.scss';
 
+export const formatVersion = (version) => `${version.major}.${version.minor}.${version.patch}`;
+
+const isOlder = (a, b) => {
+  if (a.major < b.major) {
+    return true;
+  }
+  if (a.major > b.major) {
+    return false;
+  }
+  if (a.minor < b.minor) {
```

Comment:
Do we need to safeguard here if the major is not undefined for example? It seems it is implied the major has been taken care of but what if the version is not valid

Reply:
/shrug

I copied the function over from the Versions file as this was the only place in the code where it was used